src/back/controllers/api/routes/my/my.js

```diff
@@ -14,5 +14,6 @@ function responseFactory(action) {
 module.exports = {
```

Comment:
@OblivionHammer, there're two params for these actions - user and challenge id and they are enough to perform the actions. Why should I pass more data than required?

It is also a matter of responsibility separation - I don't want action to "know" anything about `express`'s request.